packages/contracts-ops/contracts/CallBatch.sol

```diff
@@ -15,7 +17,9 @@ abstract contract CallBatch is Script {
     string public domain;
     string public outputFile;
 
-    function initializeCallBatch(
+    GovernanceMessage.Call built;
```

Comment:
in a built callbatch, i believe `to` should be an `address` not a `bytes32`

Reply:
@prestwich the `built` is copy-pasted into Gnosis multisig transactions. if the `to` is a `bytes32` then it's going to be a problem. we have multiple addresses which start with zeros, too, so cutting leading zeros isn't a good solution.

Reply:
built is not used, so it's been removed. the output writes an address (see the `vm.toString` call in `build()`